CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## Next Release
+
+- The `Message` attribute of an `APIError` is now a `interface{}` rather than a `string`, due to potential inconsistent data structure from the API response
```

Comment:
```suggestion
- The `Message` attribute of an `APIError` is now an `interface{}` rather than a `string`, due to potential inconsistent data structure from the API response
```